### PR TITLE
appveyor: add libpcre-1.dll

### LINKED ISSUE #1972: Current git version crashes on Windows when the Settings dialog is opened
Because `libpcre-1.dll` is missing, dependency of libenchant

### PR DESCRIPTION
**Pull request by un1versal**
fix #1972 (asif) take 1

### CODE REVIEW

**Comment by un1versal** [comment]
screw you jom,

**Comment by ctrlaltca** [comment]
What's the point of using jom over nmake anyway?
The free "open source" appveyor plan provides only single-core vms: https://www.appveyor.com/docs/build-configuration#virtual-machine-configurations

**Comment by un1versal** [comment]
> What's the point of using jom over nmake anyway?
> The free "open source" appveyor plan provides only single-core vms: https://www.appveyor.com/docs/build-configuration#virtual-machine-configurations

Not that this PR will work anyway, but wont a single core still be able to handle parallel jobs?

**Comment by ctrlaltca** [comment]
I don't think so; IIRC jom queries the number of cores in the box and adapt the number of running processes, so in a 1-core vm you'll get only 1 running job, to avoid losing time for context switching.

**Comment by Dessa** [comment]
normally, ncpu+1 should provide a small performance benefit, but at this point, i'll probably look into using msbuild with /MP over jom over the course of the next few days.

File: .appveyor.yml [hunk 1]
```diff
@@ -115,11 +105,8 @@ build_script:
...
+        cmake .. "-GNMake Makefiles NMAKE" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_PREFIX_PATH=c:/Qt/$env:qt_ver" "-DZLIB_INCLUDE_DIR=$zlibinc" "-DZLIB_LIBRARY=$zliblib" "-DOPENSSL_ROOT_DIR=c:/OpenSSL-$env:ssl_arch" "-DWANT_PERL=YES" "-DPERL_EXECUTABLE=c:/Strawberry/perl/bin/perl.exe" "-DWANT_PHONON=NO" "-DCMAKE_START_TEMP_FILE=" "-DCMAKE_END_TEMP_FILE=" "-DCMAKE_VERBOSE_MAKEFILE=1" "-DEnchant_FOUND=1" "-DEnchant_INCLUDE_DIRS=c:/enchant-headers;$env:mingw_root/include/enchant;$env:mingw_root/include/glib-2.0" "-DEnchant_LDFLAGS=$env:mingw_root/lib/libenchant.dll.a" "-DWANT_KDE=NO" "-DWANT_PYTHON=YES"
```
Comment (Dessa) [inline .appveyor.yml]: its "-GNMake Makefiles"

File: .appveyor.yml [hunk 2]
```diff
@@ -115,11 +105,8 @@ build_script:
...
     - ps: Push-AppveyorArtifact CMakeCache.txt
-    - jom install VERBOSE=1
```
Comment (Dessa) [inline .appveyor.yml:122]: nmake install

**Comment by un1versal** [comment]
Dessa any hints on adding libpcre properly, or shall I leave you to it?

**Comment by Dessa** [comment]
i don't have cygwin installed so the best shot that we have right now is exactly what you did.

**Comment by DarthGandalf** [comment]
When I was setting this up, jom was significantly faster than nmake. I don't know why, and how many cores it used.

**Comment by wodim** [comment]
I confirm this works but I haven't got a clue about the jom/nmake change so I am not merging it.

**Pull request merged**